src/test/java/org/ays/tests/institution/emergencyevacuationapplicationmanagementservice/PostEmergencyEvacuationApplicationsTest.java

```diff
@@ -0,0 +1,231 @@
+package org.ays.tests.institution.emergencyevacuationapplicationmanagementservice;
+
+import io.restassured.response.Response;
+import org.ays.endpoints.InstitutionEndpoints;
+import org.ays.endpoints.LandingEndpoints;
+import org.ays.payload.EmergencyEvacuationApplication;
+import org.ays.payload.ListEmergencyEvacuationApplications;
+import org.ays.payload.Orders;
+import org.ays.utility.AysResponseSpecs;
+import org.ays.utility.DataProvider;
+import org.ays.utility.DatabaseUtility;
+import org.ays.utility.ErrorMessage;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static org.hamcrest.Matchers.anyOf;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+
+public class PostEmergencyEvacuationApplicationsTest {
+    @Test(groups = {"Smoke", "Regression"})
+    public void testFilteringEvacuationApplicationsWithAnExistingApplicationData() {
+        EmergencyEvacuationApplication application = EmergencyEvacuationApplication.generateForMe();
+        LandingEndpoints.postEmergencyEvacuationApplication(application);
+        Response response = InstitutionEndpoints.postEmergencyEvacuationApplications(ListEmergencyEvacuationApplications.generate(application));
+        response.then()
+                .spec(AysResponseSpecs.expectSuccessResponseSpec())
+                .body("response.content[0].id", notNullValue())
```

Comment:
Bütün kayıtları kontrol etmek için sanırım aşağıdaki şekilde kontrol edebiliyoruz, deneyebilir misin?
```suggestion
                .body("response.content[*].id", notNullValue())
```

Reply:
@agitrubard Evet yapılabiliyor ama aslında bu testte zaten birden fazla id olmasını beklemiyoruz çünkü spesifik bir application'a gore filitreleme yapıyoruz. Yani bize ilk getirdiği ile ilgileniyoruz ama bir test case daha ekleyip filitreleme yapmadan listeleme yapacağım, o zaman senin belirttiğin gibi yapacağım.

Reply:
@agitrubard bir de restassured'da syntax response.content[*].id seklinde degil response.content*.id seklinde. Ben ayni işlevi goren baska bir assertion ile yaptım yeni eklediğim testi.

Reply:
Tamamdır, eline sağlık 🙌🏼